repose-aggregator/components/filters/compression-filter/src/main/scala/org/openrepose/filters/compression/CompressionFilter.scala

```diff
@@ -91,6 +93,7 @@ class CompressionFilter @Inject()(configurationService: ConfigurationService, co
           logger.error("Error with the CompressingFilter", e)
           httpResponse.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR)
       }
+      httpResponse.commitToResponse()
```

Comment:
weird. ok, then i agree leaving this in is reasonable so that the filter is properly self contained. I'd like to know where they failure to write is happening otherwise.